rules/employees-branding/rule.md

```diff
@@ -22,19 +20,14 @@ Also remember that uniforms don't have to be boring or uncomfortable, but they d
 
 <!--endintro-->
 
-
-::: good  
-![Figure: Good Example - Part of the SSW team attending NDC Sydney 2019 wearing special shirts we created for this event + branded backdrop for our booth](ndcshot.png)  
+::: good\
+![Figure: Good Example - Part of the SSW team attending NDC Sydney 2019 wearing special shirts we created for this event + branded backdrop for our booth](ndcshot.png)\
 :::
 
-
- ![](_C160408.jpg) 
-
-
-::: good  
-![Figure: Good Example - Some of our Sydney team wearing different branded shirt](\_C160408.jpg)  
+::: good\
```

Comment:
these trailing slashes need to be removed (it is a bug with the Netlify CMS editor, if you edit it again I think they should go away.